code/modules/mob/living/simple_animal/hostile/flan.dm

```diff
@@ -0,0 +1,66 @@
+//Will probably eventually be expanded to fit multiple types of Flan because I am a nerd.
+
+/mob/living/simple_animal/hostile/flan
+	name = "Flan"
+	desc = "Definitely not a dessert."
+	var/casting = 0
+	icon_state = "flan"
+	icon_living = "flan"
+	turns_per_move = 5
+	response_help = "pokes"
+	response_disarm = "shoves"
+	response_harm = "hits"
+	environment_smash = 0
+	speed = -2
+	maxHealth = 50
+	health = 50
+	harm_intent_damage = 5
+	melee_damage_lower = 5
+	melee_damage_upper = 10
+	attacktext = "headbutts"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	a_intent = INTENT_HARM
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	check_friendly_fire = 0			//Doesn't use projectiles, friendly fire impossible. Probably.
+	del_on_death = 1
+	ranged = 1
+	retreat_distance = 2
+	minimum_distance = 4
+	AIStatus = AI_IDLE
+	ranged_message = "begins to cast something"
+	ranged_cooldown = 25
+	var/spellname = "a Water spell!"
+	var/spellsound = 'sound/effects/spray3.ogg'
+	var/spellanimation = ATTACK_EFFECT_SMASH		//More in defines/misc.dm
+	var/spelldamagetype = BRUTE
+	var/spelldamage = 20
+
+/mob/living/simple_animal/hostile/flan/New()
+	..()
+	icon_state = "[initial(icon_state)][casting]"
```

Comment:
Is casting ever going to be 1 upon New()? If not icon_state above can just be `flan0` and this isn't needed.